packages/web3/src/web3.ts

```diff
@@ -91,6 +92,7 @@ export class Web3 extends Web3Context<EthExecutionAPI> {
 			recover: typeof recover;
 			encrypt: typeof encrypt;
 			decrypt: typeof decrypt;
+			wallet: any;
```

Comment:
Wouldn't be better to have more specific type?

Reply:
yes, was using this as a work-around to write docs but will fix it 👍  thanks